Translations for vector tiles sandbox (#4529)
opentripplanner/OpenTripPlanner: docs/sandbox/MapboxVectorTilesApi.md

      "minZoom": 12,
      "cacheMaxSeconds": 600
    },
    // all rental places: stations and free-floating vehicles
    {
      "name": "citybikes",
      "type": "VehicleRental",
      "mapper": "Digitransit",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 60,
      "expansionFactor": 0.25
    },
    // just free-floating vehicles
    {
      "name": "rentalVehicles",
      "type": "VehicleRentalVehicle",
      "mapper": "DigitransitRealtime",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 60
    },
    // just rental stations
    {
      "name": "rentalStations",
      "type": "VehicleRentalStation",
      "mapper": "Digitransit",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 600
    },
    // Contains just stations and realtime information for them
    {

>       "name": "rentalStations",

Reviewer: We have the same layer name several times in the example. Would this lead to a runtime error during startup?

If you want to keep the same name, can you make it clear in the comment that you can use either/or?

Author: Thanks for pointing this out. Forgot to changes the name for this. For vehicle parking, I added a comment that the other layer was for backwards compatibility reasons and has the same name

    {
      "name": "citybikes",
      "type": "VehicleRental",
      "mapper": "Digitransit",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 60,
      "expansionFactor": 0.25
    },
    // just free-floating vehicles
    {
      "name": "rentalVehicles",
      "type": "VehicleRentalVehicle",
      "mapper": "DigitransitRealtime",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 60
    },
    // just rental stations
    {
      "name": "rentalStations",
      "type": "VehicleRentalStation",
      "mapper": "Digitransit",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 600
    },
    // Contains just stations and realtime information for them
    {
      "name": "realtimeRentalStations",
      "type": "VehicleRentalStation",
      "mapper": "DigitransitRealtime",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 60
    },
    // note this has the same name as the example with the Digitransit mapper,
    // at some point we might want to do some clean up and rename this to
    // realtimeVehicleParking but for now this remains like this for backwards

>     // compatibility.

Reviewer: This is incorrect. The name of the layer can be freely chosen. You can have two layers with the same type but a different name.

Author: Yes the names can be freely chosen but I meant mostly for this documentation (and I guess people might use these names in practice). What should I write here?

Author: I tried rewording the comment here so it wouldn't leave too much room for misinterpretation https://github.com/opentripplanner/OpenTripPlanner/pull/4529/commits/8f6ac4a92fa8c5f22e038c7af7767d6ad442e643

Reviewer: I would either use another name in the example or make it clear that you have to choose one and cannot copy paste the entire file from here.

Author: I've updated it https://github.com/opentripplanner/OpenTripPlanner/pull/4529/commits/e1c1d6b631ef920cdf4c97a60b22afdd3902829d

      "minZoom": 12,
      "cacheMaxSeconds": 600
    },
    // all rental places: stations and free-floating vehicles
    {
      "name": "citybikes",
      "type": "VehicleRental",
      "mapper": "Digitransit",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 60,
      "expansionFactor": 0.25
    },
    // just free-floating vehicles
    {
      "name": "rentalVehicles",
      "type": "VehicleRentalVehicle",
      "mapper": "DigitransitRealtime",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 60
    },
    // just rental stations
    {
      "name": "rentalStations",
      "type": "VehicleRentalStation",
      "mapper": "Digitransit",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 600
    },
    // Contains just stations and realtime information for them
    {
      "name": "realtimeRentalStations",
      "type": "VehicleRentalStation",
      "mapper": "DigitransitRealtime",
      "maxZoom": 20,
      "minZoom": 14,
      "cacheMaxSeconds": 60
    },
    // note this has the same name as the example with the Digitransit mapper,

>     // and two layers should not have the same name in practice. At some point,

Reviewer 2: ```suggestion
    // and two layers can't have the same name in a single instance. At some point,
```
